### DISCUSSION TITLE
Welcome to Elder Launcher Discussions!

### DISCUSSION THREAD

**Discussion by itsarjunsinh** [itsarjunsinh/elder_launcher#7] [General]
## 👋 Welcome!
  I’m staring a GitHub Discussions so that people interested in Elder Launcher can connect here.
  * Ask questions you’re wondering about.
  * Share ideas.

**Comment by quxup** [comment]
Hey OP! Thank you so much for this launcher. I've alreaedy installed it on a few elderly peoples phones :)

**Reply by itsarjunsinh** [reply]
Hello @quxup,
Thank you very much for the contributing with the Estonian translation.
And thanks for the appreciation.

**Reply by Beach77** [reply]
I also want to say thank you for making this! Seems like the only updated simple launcher available on fdroid. I'm gonna install it on my dad's phone because he is very bad with technology haha and hopefully he will like it.
